models/project.go

```diff
@@ -305,3 +307,50 @@ func deleteProjectByID(e Engine, id int64) error {
 
 	return updateRepositoryProjectCount(e, p.RepoID)
 }
+
+// UpdateBoards updates boards priority for a project
+func UpdateBoards(boards []ProjectBoard) error {
+	for _, board := range boards {
+		if _, err := x.ID(board.ID).Cols("priority").Update(&board); err != nil {
+			log.Info("failed updating board priorities %s", err)
+			return err
+		}
+
+	}
+	return nil
```

Comment:
This should probably be done in a single transaction.

```suggestion
	sess := x.NewSession()
	if err := sess.Begin(); err != nil {
		return err
	}
	defer sess.Close()
	for _, board := range boards {
		if _, err := sess.ID(board.ID).Cols("priority").Update(&board); err != nil {
			log.Info("failed updating board priorities %s", err)
			return err
		}
	}
	return sess.Commit()
```